src/commands/moderation/giverole.js

```diff
@@ -0,0 +1,35 @@
+const { Command, CommandError, Constants, SwitchbladeEmbed } = require('../../')
+
+module.exports = class Giverole extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'giverole',
+      category: 'moderation',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_ROLES'], permissions: ['MANAGE_ROLES'] },
+      parameters: [{
+        type: 'member', acceptBot: true, missingError: 'commands:giverole.missingUser'
+      }, {
+        type: 'role', full: true, missingError: 'errors:invalidRole'
+      }]
+    })
+  }
+
+  async run ({ channel, member, author, t }, targetMember, role) {
+    const embed = new SwitchbladeEmbed(author)
+    await targetMember.addRole(role).then(modifiedMember => {
```

Comment:
Whenever you use async/await, you don't have to use `.then` or `.catch` you can use the simple try/catch. Ex.:
```js
try {
  await targetMember.addRole(role)
  // do other things
} catch (e) {
  // handle error
}
```

Reply:
Just sent this change